quickstart.rst

```diff
@@ -1,6 +1,12 @@
-**********
-User guide
-**********
+*****************
+Quick-start guide
```

Comment:
From a bit of googling I think either "Quickstart" or "Quick start" is commonly used, but not the hyphenated form.